src/components/ProductCard.jsx

```diff
@@ -1,14 +1,17 @@
 import "./ProductCard.css";
 import noImage from "../assets/noImage.jpg";
+import { useNavigate } from "react-router-dom";
 
 /**
  *  단일 상품 정보를 카드 형태로 표시하는 컴포넌트 입니다.
  *  상품 이미지, 상품명, 가격, 좋아요 수를 표시합니다.
  * 이미지가 없거나 에러가 발생하는 경우, 대체 이미지를 처리할 수 있도록 onError를 사용하였습니다.
  */
-const ProductCard = ({ item }) => {
+const ProductCard = ({ item, variant }) => {
+  const nav = useNavigate();
+  const isBest = variant === "best";
   return (
-    <div className="product-card">
+    <div className={`product-card ${isBest ? "product-card--best" : "product-card--list"}`} onClick={() => nav(`/items/${item.id}`)}>
```

Comment:
💊 제안
Link 태그가 더 시멘틱하고 다른 코멘트에서 언급한 것처럼 UX측면에서도 더 좋을 것 같아요~
```suggestion
    <Link className={`product-card ${isBest ? "product-card--best" : "product-card--list"}`} to={`/items/${item.id}`}>
```